src/main/steps/applicant/task-list/content.ts

```diff
@@ -169,26 +179,32 @@ const cy = () => ({
     {
       label: 'Ychwanegu cynrychiolydd cyfreithiol',
       link: APPLICANT_ADD_LEGAL_REPRESENTATIVE,
+      target: '',
     },
     {
       label: 'Dileu cynrychiolydd cyfreithiol',
       link: APPLICANT_REMOVE_LEGAL_REPRESENTATIVE_START,
+      target: '',
```

Comment:
dont have it empty. if target not specified then it should be treated as_self in template